contracts/sources/obligation.move

```diff
@@ -319,6 +399,14 @@ module suilend::obligation {
             deposit.deposited_ctoken_amount,
             clock
         );
+
+        if (is_emode) {
+            assert!(vector::length(&obligation.deposits) == 1, EIsolatedAssetViolation);
```

Comment:
IsolatedAsset refers to something else, so i would use `EInvalidEModeDeposit`

Reply:
Changed